app/controllers/katello/api/v2/packages_controller.rb

```diff
@@ -2,6 +2,9 @@ module Katello
   class Api::V2::PackagesController < Api::V2::ApiController
     apipie_concern_subst(:a_resource => N_("a package"), :resource => "packages")
     include Katello::Concerns::Api::V2::RepositoryContentController
+    include Katello::Concerns::Api::V2::RepositoryDbContentController
+
+    before_filter :find_repositories, :only => :auto_complete_name
 
     api :GET, "/packages", N_("List packages")
     api :GET, "/repositories/:repository_id/packages", N_("List packages")
```

Comment:
I know we have discussed balancing package vs RPM as part of this change. Should we put an eye to the future and introduce `/rpms` based APIs? And/or start the deprecation cycle for things with `packages` ?